lib/recording/create_recordig_job.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Onvif.Recording.CreateRecordingJob do
+  import SweetXml
+  import XmlBuilder
+  require Logger
+
+  def soap_action, do: "http://www.onvif.org/ver10/recording/wsdl/CreateRecordingJob"
+
+  def request(device, args) do
+    Onvif.Recording.request(device, args, __MODULE__)
+  end
+
+  def request_body(recording_token, priority \\ "0", mode \\ "Active") do
+    element(:"s:Body", [
+      element(:"trc:CreateRecordingJob", [
+        element(:"trc:JobConfiguration", [
+          element(:"tt:RecordingToken", recording_token),
+          element(:"tt:Mode", mode),
+          element(:"tt:Priority", priority)
+        ])
+      ])
+    ])
+  end
+
+  def response(xml_response_body) do
+    parsed_result =
+      xml_response_body
+      |> parse(namespace_conformant: true, quiet: true)
+      |> xpath(
+        ~x"//s:Envelope/s:Body/trc:CreateRecordingJobResponse"
+        |> add_namespace("s", "http://www.w3.org/2003/05/soap-envelope")
+        |> add_namespace("trc", "http://www.onvif.org/ver10/recording/wsdl"),
+        job_token: ~x"//trc:JobToken/text()"so,
+        recording_token: ~x"//trc:JobConfiguration/tt:RecordingToken/text()"so,
+        mode: ~x"//trc:JobConfiguration/tt:Mode/text()"so,
+        priority: ~x"//trc:JobConfiguration/tt:Priority/text()"so
+      )
```

Comment:
Shouldn't this be parsed / structified with the `RecordingJob` module?

Reply:
this is very specific for this case, the schema is also different. This information is available on other API calls, this is the only one that I was trying to do differently, but if you rather have a standard "austin model" approach, I chan revert to ths standard one

Reply:
The parsing looks exactly the same to me in the code (with the exception that the namespace for JobConfiguration is different), except you aren't pulling the `source` out here. But it's optional so the parsing should still work if it is present, and then you get a nice struct as the result

Reply:
Possibly the namespace difference would cause issues here, but if you could test it just so we know and add a note that the ONVIF api spec has a bug that would be nice